integration-tests/integration.test.js

```diff
@@ -1,299 +1,151 @@
-const { getByTestId, getByRole, fireEvent, waitFor } = require('@testing-library/dom')
-const { start, registerHtml } = require('../src/tram-one')
-
-const { frozenCounter, globalCounter, graphic, home, loadingPage, loading, removablePage, updatablePage, updatable, urlLabel } = require('./mock-components')
-
-const html = registerHtml({
-	globalCounter, home, loadingPage, loading, updatable, urlLabel
-})
+const { getByText, fireEvent, waitFor } = require('@testing-library/dom')
+const { startApp } = require('./test-app')
```

Comment:
Rather than importing and building a bunch of isolated components, we have a mock app that exercises tram-one in a variety of ways.

Reply:
As a result of this refactoring, all the mock-components made for testing have been removed, favoring instead a single app with a fewer number of components.